src/graphics.c

```diff
@@ -68,10 +69,21 @@ static inline void plot_pixel(int x, int y, struct graphics_color c) {
         mouse_draw_buffer[index].r = c.r;
         mouse_draw_buffer[index].g = c.g;
         mouse_draw_buffer[index].b = c.b;
-        graphics_draw_mouse();
+        graphics_draw_pointer_mouse();
     }
 }
 
+// like plot_pixel, but does not do mouse erasing and redrawing or bounds checking
+static inline void draw_pixel(int x, int y, struct graphics_color c) {
```

Comment:
Might just be picky, but is there a better name we can use than `draw_pixel` since it seems fairly ambiguous with `plot_pixel` especially when used throughout the file.